docs/topics/Threads.md

```diff
@@ -163,24 +163,42 @@ Additionally, when a client loses access to a channel they are not removed from
+
+Threads can be created using the [`POST /channels/<channel_id>/threads`](#DOCS_RESOURCES_CHANNEL/start-thread-in-forum-channel) endpoint. When creating threads in forum channels, the API will *also* create a message that will have the same ID as the thread. Since a message is being created, you're required to pass parameters for both a message and a thread when calling the create thread endpoint.
+
+Threads in a forum channel have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with one exception: creating a thread in a forum channel only requires the `SEND_MESSAGES` permission.
+
+### Forum Channel Fields
+
+It's worth calling out a few details about fields specific to forum channels that may be important to keep in mind:
+
+- The `last_message_id` field is the ID of the most recently created thread in that channel. As with messages, your app will not receive a `CHANNEL_UPDATE` event when the field is changed. Instead clients should update the value when receiving [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) events.
+- The `topic` field is what is shown in the "Guidelines" section within the Discord client.
+- The `rate_limit_per_user` field limits how frequently threads can be created. There is a new `default_thread_rate_limit_per_user` field on forums as well, which limits how often messages can be sent _in a thread_. This field is copied into `rate_limit_per_user` on the thread at creation time.
+- The `available_tags` field can be set when creating or updating a channel, which determines which tags can be set on individual threads within the thread's `applied_tags` field.
+
+All fields for channels, including forum channels, can be found in the [Channel Object](#DOCS_RESOURCES_CHANNEL/channel-object). 
+
+### Forum Channel Thread Fields
+
+A thread can be pinned within a forum, which will be represented as the [`PINNED` flag](#DOCS_RESOURCES_CHANNEL/channel-object-channel-flags) in the `flags` field within a forum channel. A thread that is pinned will have the `(1 << 1)` flag set, and archiving that thread will unset the flag. A pinned thread will *not* auto-archive.
+
+The `message_count` and `total_message_sent` fields on threads in forum channels will increment on `MESSAGE_CREATE` events, and decrement on `MESSAGE_DELETE` and `MESSAGE_DELETE_BULK` events. There will be no specific `CHANNEL_UPDATE` event that notifies your app of changes to those fields—instead, apps should update those values when receiving corresponding events.
+
+All fields for threads in forum channels can be found in the [channel resources documentation](#DOCS_RESOURCES_CHANNEL/start-thread-in-forum-channel-jsonform-params).
+
+### Forum Channel Thread Formatting
+
+Unlike threads in text channels, threads in forum channels can contain additional markdown for bulleted lists and headings.
```

Comment:
This only applies to the first message in the thread btw, the one that you create in the API call to make a thread.